docs/development/local-development.md

```diff
@@ -144,21 +144,21 @@ If this is working then in future you can create other test repos to verify your
 
 You can run `pnpm test` locally to test your code.
 We test all PRs using the same tests, run on GitHub Actions.
-`pnpm test` runs an `eslint` check, a `prettier` check, a `type` check and then all the unit tests using `jest`.
+`pnpm test` runs an `eslint` check, a `prettier` check, a `type` check and then all the unit tests using `vitest`.
 
 Refactor PRs should ideally not change or remove tests (adding tests is OK).
 
-### Jest
+### Vitest
 
-Run the Jest unit tests with the `pnpm jest` command.
-You can also run a subset of the Jest tests using file matching, e.g. `pnpm jest composer` or `pnpm jest workers/repository/update/branch`.
+Run the Vitest unit tests with the `pnpm vitest` command.
+You can also run a subset of the Vitest tests using file matching, e.g. `pnpm vitest composer` or `pnpm vitest workers/repository/update/branch`.
 If you get a test failure due to a "snapshot" mismatch, and you are sure that you need to update the snapshot, then you can append `-u` to the end.
-e.g. `pnpm jest composer -u` would update the saved snapshots for _all_ tests in `**/composer/**`.
+e.g. `pnpm vitest composer -u` would update the saved snapshots for _all_ tests in `**/composer/**`.
 
 ### Coverage
 
 The Renovate project maintains 100% test coverage, so any Pull Request will fail if it does not have full coverage for code.
-Using `// istanbul ignore` is not ideal, but can be a pragmatic solution if adding more tests wouldn't really prove anything.
+Using `// v0 ignore` is not ideal, but can be a pragmatic solution if adding more tests wouldn't really prove anything.
```

Comment:
```suggestion
Using `// v8 ignore` is not ideal, but can be a pragmatic solution if adding more tests wouldn't really prove anything.
```

Typo? The Vitest docs say `v8 ignore` is the right term, you're using a zero. 😉 

[Vitest docs, ignoring code](https://vitest.dev/guide/coverage.html#ignoring-code)